timetracker/models.py

```diff
@@ -1,31 +1,43 @@
 from django.db import models
 from django.utils import timezone
+from datetime import date
 
 
 # Create your models here.
 
 class Employee(models.Model):
-    name = models.CharField(max_length=50)
+    name = models.CharField(max_length=50, blank=False)
 
     def __str__(self):
         return self.name
 
+
 class Checking(models.Model):
```

Comment:
update this model to have 2 datetime columns, one for checkin and the other for checkout. Then all the calculations in the API can be done within the querysets.